### PR TITLE
Update sp-compatible-pdf-readers-for-irm.md

### PR DESCRIPTION
**Pull request by msbemba**
Changed Foxit downloadable link(S)

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2521

File: microsoft-365/compliance/sp-compatible-pdf-readers-for-irm.md [hunk 1]
```diff
@@ -27,11 +27,11 @@ To use PDF files in libraries that the owner has protected with IRM, the user wi
...
+|Windows Vista  <br/> |Foxit Reader  <br/> NitroPDF  <br/> |[Download Foxit Reader](https://www.foxitsoftware.com/downloads/#Foxit-PhantomPDF-Business/) <br/> [Download NitroPDF](https://www.gonitro.com/pdf-reader) <br/> |
+|Windows 7  <br/> |Azure Information Protection app  <br/> Foxit Reader  <br/> NitroPDF  <br/> |[Download Azure Information Protection app](https://go.microsoft.com/fwlink/?linkid=837797) <br/> [Download Foxit Reader](https://www.foxitsoftware.com/downloads/#Foxit-PhantomPDF-Business/) <br/> [Download NitroPDF](https://www.gonitro.com/pdf-reader) <br/> |
+|Windows 8 (classic mode)  <br/> |Azure Information Protection app  <br/> Foxit Reader  <br/> NitroPDF  <br/> |[Download Azure Information Protection app](https://go.microsoft.com/fwlink/?linkid=837797) <br/> [Download Foxit Reader]((https://www.foxitsoftware.com/downloads/#Foxit-PhantomPDF-Business/) <br/> [Download NitroPDF](https://www.gonitro.com/pdf-reader) <br/> |
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/sp-compatible-pdf-readers-for-irm.md]:
```suggestion
|Windows 8 (classic mode)  <br/> |Azure Information Protection app  <br/> Foxit Reader  <br/> NitroPDF  <br/> |[Download Azure Information Protection app](https://go.microsoft.com/fwlink/?linkid=837797) <br/> [Download Foxit Reader](https://www.foxitsoftware.com/downloads/#Foxit-PhantomPDF-Business/) <br/> [Download NitroPDF](https://www.gonitro.com/pdf-reader) <br/> |
```

File: microsoft-365/compliance/sp-compatible-pdf-readers-for-irm.md [hunk 2]
```diff
@@ -27,11 +27,11 @@ To use PDF files in libraries that the owner has protected with IRM, the user wi
...
+|Windows 8.1  <br/> |Azure Information Protection app  <br/> Foxit Reader  <br/> NitroPDF  <br/> |[Download Azure Information Protection app](https://go.microsoft.com/fwlink/?linkid=837797) <br/> [Download Foxit Reader]((https://www.foxitsoftware.com/downloads/#Foxit-PhantomPDF-Business/) <br/> [Download NitroPDF](https://www.gonitro.com/pdf-reader) <br/> |
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/sp-compatible-pdf-readers-for-irm.md]:
```suggestion
|Windows 8.1  <br/> |Azure Information Protection app  <br/> Foxit Reader  <br/> NitroPDF  <br/> |[Download Azure Information Protection app](https://go.microsoft.com/fwlink/?linkid=837797) <br/> [Download Foxit Reader](https://www.foxitsoftware.com/downloads/#Foxit-PhantomPDF-Business/) <br/> [Download NitroPDF](https://www.gonitro.com/pdf-reader) <br/> |
```

File: microsoft-365/compliance/sp-compatible-pdf-readers-for-irm.md [hunk 3]
```diff
@@ -27,11 +27,11 @@ To use PDF files in libraries that the owner has protected with IRM, the user wi
...
+|Windows 10  <br/> |Azure Information Protection app  <br/> Foxit Reader  <br/> NitroPDF  <br/> |[Download Azure Information Protection app](https://go.microsoft.com/fwlink/?linkid=837797) <br/> [Download Foxit Reader]((https://www.foxitsoftware.com/downloads/#Foxit-PhantomPDF-Business/) <br/> [Download NitroPDF](https://www.gonitro.com/pdf-reader) <br/> |
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/sp-compatible-pdf-readers-for-irm.md]:
```suggestion
|Windows 10  <br/> |Azure Information Protection app  <br/> Foxit Reader  <br/> NitroPDF  <br/> |[Download Azure Information Protection app](https://go.microsoft.com/fwlink/?linkid=837797) <br/> [Download Foxit Reader](https://www.foxitsoftware.com/downloads/#Foxit-PhantomPDF-Business/) <br/> [Download NitroPDF](https://www.gonitro.com/pdf-reader) <br/> |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@KCCross Please merge this PR. Thanks

**Comment by KCCross** [comment]
Folks who own the original fwlink are no longer with the company. Created new fwlink. Also asked for ownership of the old fwlink to be transferred to the pubs security group for fwlinks just in case it exists elsewhere.

**Comment by KCCross** [comment]
We also created 2139327 for Nitro PDF reader.

**Comment by msbemba** [comment]
created a new PR # https://github.com/MicrosoftDocs/microsoft-365-docs/pull/2631

**Pull request merged**